editor.planx.uk/src/api/upload.ts

```diff
@@ -6,16 +6,18 @@ async function uploadFile(
   file: any,
   { onProgress }: { onProgress?: (p: any) => void } = {}
 ) {
-  const res = await fetch(`${process.env.REACT_APP_API_URL}/sign-s3-upload`, {
-    method: "POST",
-    body: JSON.stringify({
+  const res = await axios.post(
+    `${process.env.REACT_APP_API_URL}/sign-s3-upload`,
+    JSON.stringify({
       filename: file.name,
     }),
-    headers: {
-      "Content-Type": "application/json",
-    },
-  });
-  const token = await res.json();
+    {
+      headers: {
+        "Content-Type": "application/json",
+      },
+    }
+  );
+  const token = JSON.parse(res.data);
```

Comment:
I know this is still a draft, so apologies if overstepping, but I was peaking through initials changes and realized I might have failed to explain something here! 

The way that we send files to Uniform is not via S3, which is what this file/endpoint handles (for user-uploaded files specifically through the FileUpload component), but rather by generating a temporary zip folder locally and attaching that to Idox's Submission API (which unfortunately does not have normal internet documentation I can link, but there is a Postman Collection you can find in 1Password that's the closest thing we have to docs from the Idox team!). 

On the frontend, it might be helpful to look at `getUniformParams` which [returns](https://github.com/theopensystemslab/planx-new/blob/main/editor.planx.uk/src/%40planx/components/Send/uniform/index.ts#L42-L48) a general outline of files that Uniform will require. Then, follow that through to [this API file](https://github.com/theopensystemslab/planx-new/blob/main/api.planx.uk/send/uniform.js), which outlines how we generate files from those params and actually submit them to Uniform! 

Of course always many ways to do the thing and improvements very welcome - but just want to clear up any S3 file upload versus local file stuff early! Please just shout if anything still isn't clear!